Mlem/App/Globals/Definitions/AppState+transition.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  AppState+transition.swift
```

Comment:
*nit*

Capitalization

```suggestion
//  AppState+Transition.swift
```